sqlserver/datadog_checks/sqlserver/sqlserver.py

```diff
@@ -533,19 +533,28 @@ def do_stored_procedure_check(self, instance, proc):
         """
 
         guardSql = instance.get('proc_only_if')
+        custom_tags = instance.get("tags", [])
 
         if (guardSql and self.proc_check_guard(instance, guardSql)) or not guardSql:
             self.open_db_connections(instance, self.DEFAULT_DB_KEY)
             cursor = self.get_cursor(instance, self.DEFAULT_DB_KEY)
 
             try:
                 self.log.debug("Calling Stored Procedure : {}".format(proc))
-                cursor.callproc(proc)
+                if self._get_connector(instance) == 'adodbapi':
+                    cursor.callproc(proc)
```

Comment:
This is unrelated to the custom tags changes. Let's do that in a separate PR please.

Reply:
Okay I merged #3237 so you should be able to rebase on master and have only your custom tags changes here